mureader.py

```diff
@@ -1,23 +1,24 @@
 from struct import pack, unpack
 
 def get_mu_textures(mu_path):
-	''' Returns the list of the referenced texture paths'''
-        self.textures = []
-        with open(mu_path, "rb") as self.file:
-            self.magic, self.version = self.read_int(2)
-            if (self.magic != MuEnum.MODEL_BINARY or self.version < 0
-                or self.version > MuEnum.FILE_VERSION):
-                return None
-            self.name = self.read_string()
-            self.obj = MuObject().read(self)
-            #self.read_materials()
-            #self.read_textures()
-            del self.file
-            return self
+    ''' Returns the list of the referenced texture paths'''
+    self.textures = []
```

Comment:
this is not our code, right? I don't understand what `self` is here. If this just a 'cosmit' (as it seems to be), I'd rather have a separate commit to make hanges easier to track and avoid later conflicts; maybe we should not even change this if it doesn't break anything